packages/eas-cli/src/utils/profiles.ts

```diff
@@ -79,15 +83,21 @@ async function readProfileAsync<T extends ProfileType>({
 
 let hasPrintedDeprecationWarnings = false;
 
-function maybePrintBuildProfileDeprecationWarnings(
+export function clearHasPrintedDeprecationWarnings(): void {
+  hasPrintedDeprecationWarnings = false;
+}
```

Comment:
This is only necessary for tests? Maybe add a comment it shouldn't be used elsewhere.